src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -4360,6 +4382,24 @@ subroutine atm_compute_dyn_tend_work(nCells, nEdges, nVertices, nVertLevels_dumm
             h_theta_eddy_visc4 = config_h_theta_eddy_visc4
 
          end if
+
+         if(config_horiz_absorbing_layer == "mpas_cam") then
+
+            do iCell = cellStart,cellEnd
+
+            !
+            ! 2nd-order filter for top absorbing layer as in CAM-SE :  WCS 10 May 2017
+            ! From MPAS-CAM V4.0 code, with addition to config-specified coefficient (V4.0_coef = 0.2; SE_coef = 1.0)
+            !
+            kdiff(nVertLevels-2,iCell) = max(kdiff(nVertLevels-2,iCell),   2.0833*config_len_disp*config_mpas_cam_coef)
+            kdiff(nVertLevels-1,iCell) = max(kdiff(nVertLevels-1,iCell),2.*2.0833*config_len_disp*config_mpas_cam_coef)
+            kdiff(nVertLevels  ,iCell) = max(kdiff(nVertLevels  ,iCell),4.*2.0833*config_len_disp*config_mpas_cam_coef)
```

Comment:
Where do the `2.0833` values come from?

Reply:
This coefficient comes from CAM - will check and include in documentation.